config/templates/stack-template.json

```diff
@@ -17,19 +17,19 @@
           "{{$subnet.AvailabilityZone}}"
           {{end}}
         ],
-        "DesiredCapacity": "{{.WorkerCount}}",
+        "DesiredCapacity": "{{.WorkerASG.MinSize}}",
```

Comment:
My assumption is desired capacity is usually min size for cluster rollout? Afterwards AWS changes the desired capacity when we implement auto scaling. We could allow this to be customised as well? I can't remember the AWS behaviour if desired capacity is outside min/max etc. Right now I just replaced like for like so we could leave the extra config for later, i.e. it was the min count before and it is still the min count.